test/localstack.ts

```diff
@@ -125,6 +127,8 @@ export const localstackTestSuite = (
       return {};
     });
 
+    await Promise.all(Function.promises);
```

Comment:
`deployStack` calls `asyncSynth` which awaits on the function promises. Was something not working?